src/main/java/edu/cornell/kfs/module/purap/document/CuRequisitionDocument.java

```diff
@@ -435,7 +440,11 @@ public void prepareForSave(KualiDocumentEvent event) {
     public Date getCreateDateForResult() {
         return getCreateDate();
     }
-        
+
+    protected CuPurapService getPurapService() {
+        return SpringContext.getBean(CuPurapService.class);
```

Comment:
I had declared this method as "protected" so that a unit-test-specific subclass could override it if necessary. If you think an explicit field would be better, please let me know.